### ISSUE TITLE
I PAID BUT I DONT RECEIVE

### ISSUE THREAD

**Issue by Peaceelove** [dalaranwow/dalaran-wow#5259]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
at 9:51 pm on 14/01/2018 I paid 10 euros to buy 10 donation points, but after three hours I have not received anything yet. 10 euros, however, I was correctly deducted from my card.
**Expected behaviour**: Tell us what should happen instead
After the purchase I should have received my 10 donation points.
**Steps to reproduce the problem**:

1. I wanted to buy donation points to be able to change factions
2. At 9:51 pm on 14/01/2018 I paid 10 euros to get 10 donation points, i bought the donation points correctly and they have been deducted from the card I used to pay
3. After 3 hours i still not receive anything

**Include proofs for this behaviour**
![cattura](https://user-images.githubusercontent.com/35436475/34922289-43d2811a-f98e-11e7-853d-3bbcdf6756bd.PNG)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
